### DISCUSSION TITLE
[Feature request] arm64 .NET support

### DISCUSSION THREAD

**Discussion by flanter21** [kwsch/PKHeX#4130] [Ideas]
**Is your feature request related to a problem? Please describe.**
I run PKHeX through a windows virtual machine on my mac. Upon downloading the latest release, I was prompted to update my .NET version. Noticing it was downloading the x64 version automatically, I clicked around and downloaded the arm64 version instead. Unfortunately PKHeX still did not run. It ran only after using the x64 version.

**Describe the solution you'd like**
PKHeX detecting arm64 .NET or arm64 binary/compile instructions provided.

**Describe alternatives you've considered**
Using x64 .NET runtime works very well. I only opened this discussion in case it is trivial to implement and has been overlooked.

**Additional context**
N/A thank you for providing such a well-built program

**Comment by kwsch** [comment]
PKHeX isn't CPU architecture specific, but remember it is a Windows Forms application, so if you're using a Mac, you'll need to use something like Wine to provide a compatibility layer.
The program is published as 64bit for general consumption, but anyone can compile it with whatever supported CPU architecture they want.
